projects/fourier/lib/fourier/utilities/system.rb

```diff
@@ -9,17 +9,17 @@ def self.system(*args)
         Kernel.system(*args) || exit(1)
       end
 
-      def self.tuist(*args, **kwargs)
-        @tuist = "/usr/local/bin/tuist"
+      def self.tuist(*args, source: false)
+        tuist = "/usr/local/bin/tuist"
```

Comment:
There's no need to hardcode the path here. `/usr/local/bin` is the default, but it could be installed elsewhere.

```suggestion
        tuist = "tuist"
```